managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "authors"
+
+    def create(self, authors: str):
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (format) VALUES (?)",
+            (format_,)
```

Comment:
The SQL statement in the `create` method is incorrect. It should insert into `first_name` and `last_name` columns, not `format`. Update the SQL statement to reflect the correct columns.